SAGESharp/SLB/Cinematic/Camera/Camera.cs

```diff
@@ -0,0 +1,105 @@
+using SAGESharp.SLB.IO;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SAGESharp.SLB.Cinematic.Camera
+{
+    public sealed class Camera : IEquatable<Camera>
+    {
+
+
+
+        internal const int BINARY_SIZE = 20; //TODO: I don't know what I should count
+
+        [SLBElement(1)]
+        public float ViewAngle { get; set; }
+
+        [SLBElement(2)]
+        public float SpinMaskTimes1 { get; set; }
+
+        [SLBElement(3)]
+        public float SpinMaskTimes2 { get; set; }
+
+        [SLBElement(4)]
+        public float SpinMaskTimes3 { get; set; }
+
+        [SLBElement(5)]
+        public IList<Frame> Frames { get; set; }
+
+        public bool Equals(Camera other)
+        {
+            if (other == null)
+            {
+                return false;
+            }
+
+            return ViewAngle == other.ViewAngle &&
+                SpinMaskTimes1 == other.SpinMaskTimes1 &&
+                SpinMaskTimes2 == other.SpinMaskTimes2 &&
+                SpinMaskTimes3 == other.SpinMaskTimes3 &&
+                Frames.SafeSequenceEquals(other.Frames);
+        }
+
+        public override string ToString()
```

Comment:
Use string interpolation:

```
return $"ViewAngle={ViewAngle}," +
    $"SpinMaskTimes1={SpinMasktimes1}," +
    $"SpinMaskTimes2={SpinMasktimes2}," +
    $"SpinMaskTimes3={SpinMasktimes3}," +
    $"Frames={Frames?.Let(frames => "[(" + string.Join("), (") + ")]") ?? "null"}";
```

You will also need to add: `using Konvenience;` at the top

Reply:
I missed here a chunk, the last line should be:

```
    $"Frames={Frames?.Let(frames => "[(" + string.Join("), (", frames) + ")]") ?? "null"}";
```